server/sonar-web/src/main/js/apps/issues/components/BulkChangeModal.js

```diff
@@ -144,31 +148,33 @@ export default class BulkChangeModal extends React.PureComponent {
     }
   };
 
-  handleAssigneeSelect = (assignee: string) => {
+  handleAssigneeSelect = (assignee /*: string */) => {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)